boards/x86/quark_se_c1000_devboard/dts_fixup.h

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
why do we have this as part of the board dts fixup? the board does not have this module, it is an add-on that can be added to ANY board.

Reply:
Oops, I should move this down into the sample :)